tests/rgw/sanity_rgw.py

```diff
@@ -95,6 +98,12 @@ def run(ceph_cluster, **kw):
     lib_dir = DIR[test_version]["lib"]
     timeout = config.get("timeout", 300)
 
+    if test_config["config"]:
+        log.info("creating custom config")
+        f_name = test_folder + config_dir + config_file_name
+        remote_fp = rgw_node.remote_file(file_name=f_name, file_mode="w")
+        remote_fp.write(yaml.dump(test_config, default_flow_style=False))
+
```

Comment:
I strongly believe we should start decoupling the target node and allow the user the control the target.

Reply:
agree, so i am planning to make some changes to `DBR`. i will try to decouple then.